### DISCUSSION TITLE
能否提供类似 Vercel 或者 Netlify 那样的一键部署按钮？

### DISCUSSION THREAD

**Discussion by Yidadaa** [zeabur/.github#7] [Q&A] [answered]
类似这样：

<img width="172" alt="image" src="https://user-images.githubusercontent.com/16968934/231419045-afbad46b-b276-4908-88c9-9d0c10b34474.png">

点击之后就可以一键导入 Github 仓库并部署，这对于开源项目来说非常有用。

**Comment by MichaelYuhe** [comment]
有在计划中了～感谢反馈

**Comment by MichaelYuhe** [comment, marked as answer]
<img width="443" alt="image" src="https://github.com/zeabur/.github/assets/63531512/e21881cf-d7d8-4363-8771-0bc2f109a12e">

1. Create a template in Dashboard: https://dash.zeabur.com/create-template
2. Copy the button as HTML or Markdown: https://dash.zeabur.com/account/template
